cmd/agent/common/helpers.go

```diff
@@ -86,19 +86,19 @@ func SelectedCheckMatcherBuilder(checkNames []string, minInstances uint) func(co
 // SetupInternalProfiling is a common helper to configure runtime settings for internal profiling.
 func SetupInternalProfiling(cfg config.ConfigReader, configPrefix string) {
 	if v := cfg.GetInt(configPrefix + "internal_profiling.block_profile_rate"); v > 0 {
-		if err := settings.SetRuntimeSetting("runtime_block_profile_rate", v); err != nil {
+		if err := settings.SetRuntimeSetting("runtime_block_profile_rate", v, settings.LogLevelSourceCLI); err != nil {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ Question:** ‏Is it actually CLI everywhere? If not or if we're not sure, we should either put `default` or `unknown`. Or we add another source for `app` or something.


Comment applies to every runtime setting that's not log level

Reply:
I think this function is triggered when you enter `./bin/agent/agent config set log_level XXX -c ./dev/dist/datadog_staging.yaml`
So we definitely need to set the source to `CLI` here

However it might be used elsewhere too...
No idea how to ensure that this doesn't lead to sneaky issues

Reply:
Nope I was wrong, we found ou IRL that the change happens in 
`datadog-agent/pkg/config/settings/http/server.go`